lib/srv/uacc/uacc_linux.go

```diff
@@ -212,3 +231,14 @@ func UserWithPtyInDatabase(utmpPath string, username string) error {
 		return nil
 	}
 }
+
+// check if running libcmusl
+func isLibCMusl() bool {
```

Comment:
My knee jerk feeling is that this might be too frequent on each utmp/wtmp API invocation for no good reason.
I'd be in favor of detecting musl at init once or putting wtmp/utmp support behind a configuration option which can be switched off.